core/src/verified_vote_packets.rs

```diff
@@ -185,6 +187,13 @@ impl SingleValidatorVotes {
         }
     }
 
+    fn get_latest_gossip_timestamp(&self) -> Option<UnixTimestamp> {
```

Comment:
nit: think we can just name this to `get_timestamp`, since it's not necessarily from gossip